packages/devextreme/scss/widgets/fluent/loadIndicator/_colors.scss

```diff
@@ -9,3 +9,4 @@
 * $type color
 */
 $load-indicator-segment-bg-color: $base-accent !default;
+$load-indicator-inner-segment-border-color: #D0E5FB !default;
```

Comment:
What about dark theme? The colors of segments are incorrect there

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/25914/commits/b7809f549dc34382eddd031f028ffdbd38857d0c